samples/cut-geometry/src/main/java/com/esri/arcgismaps/sample/cutgeometry/components/CutGeometryViewModel.kt

```diff
@@ -0,0 +1,215 @@
+            addPoint(Point(-9605207.742329, 6206654.660191))
+            addPoint(Point(-9564786.389509, 6108834.986367))
+            addPoint(Point(-9449989.747500, 6095091.726408))
+            addPoint(Point(-9462116.153346, 6044160.821855))
+            addPoint(Point(-9417652.665244, 5985145.646738))
+            addPoint(Point(-9438671.768711, 5946341.148031))
+            addPoint(Point(-9398250.415891, 5922088.336339))
+            addPoint(Point(-9419269.519357, 5855797.317714))
+            addPoint(Point(-9467775.142741, 5858222.598884))
+            addPoint(Point(-9462924.580403, 5902686.086985))
+            addPoint(Point(-9598740.325877, 5884092.264688))
+            addPoint(Point(-9643203.813979, 5845287.765981))
+            addPoint(Point(-9739406.633691, 5879241.702350))
+            addPoint(Point(-9783061.694736, 5922896.763395))
+            addPoint(Point(-9844502.151022, 5936640.023354))
+            addPoint(Point(-9773360.570059, 6019099.583107))
+            addPoint(Point(-9883306.649729, 5968977.105610))
+            addPoint(Point(-9957681.938918, 5912387.211662))
+            addPoint(Point(-10055501.612742, 5871965.858842))
+            addPoint(Point(-10116942.069028, 5884092.264688))
+            addPoint(Point(-10111283.079633, 5933406.315128))
+            addPoint(Point(-10214761.742852, 5888134.399970))
+            addPoint(Point(-10254374.668616, 5901877.659929))
+        }.toGeometry()
+    }
+
+    // create a polyline that divides Lake Superior into a Canada side and US side
+    private val borderPolyline by lazy {
+        PolylineBuilder(SpatialReference.webMercator()) {
+            addPoint(Point(-9981328.687124, 6111053.281447))
+            addPoint(Point(-9946518.044066, 6102350.620682))
+            addPoint(Point(-9872545.427566, 6152390.920079))
+            addPoint(Point(-9838822.617103, 6157830.083057))
+            addPoint(Point(-9446115.050097, 5927209.572793))
+            addPoint(Point(-9430885.393759, 5876081.440801))
+            addPoint(Point(-9415655.737420, 5860851.784463))
+        }.toGeometry()
+    }
+
+    // create a blue polygon graphic to represent lake superior
+    private val polygonGraphic = Graphic(
+        lakeSuperiorPolygon,
+        SimpleFillSymbol(
+            SimpleFillSymbolStyle.Solid, Color(R.color.transparentBlue),
+            SimpleLineSymbol(SimpleLineSymbolStyle.Solid, Color.blue, 2F)
+        )
+    )
+
+    // create a red polyline graphic to represent the cut line
+    private val polylineGraphic = Graphic(
+        borderPolyline, SimpleLineSymbol(
+            SimpleLineSymbolStyle.Dot,
+            Color.red, 3F
+        )
+    )
+
+    private val _isResetButtonEnabled = MutableStateFlow(false)
+    val isResetButtonEnabled = _isResetButtonEnabled.asStateFlow()
+
+    private val _isCutButtonEnabled = MutableStateFlow(false)
+    val isCutButtonEnabled = _isCutButtonEnabled.asStateFlow()
```

Comment:
Could add a comment explaining what these do.

I was wondering if these were simple enough they could be `remember{ mutableStateOf(true/false) }` in the main screen. I think waiting for the map to load before cut enables makes it just complex enough to not be worth it.